src/components/TodoList/TodoList.jsx

```diff
@@ -1,4 +1,9 @@
-// Add the required props
-export const TodoList = () => (
-  <>TodoList markup</>
+import { TodoInfo } from '../TodoInfo/TodoInfo';
+
+export const TodoList = ({ todos }) => (
+  <section className="TodoList">
+    {todos.map(todo => (
```

Comment:
No check for empty 'todos'. Before mapping over 'todos', it would be a good practice to check if 'todos' is not empty. If it is empty, you could return a message saying that there are no todos.